### PR TITLE
master-lps-136811

### PR DESCRIPTION
**Pull request by LeandroTravassos**
https://issues.liferay.com/browse/LPS-136811

### CODE REVIEW

**Comment by LeandroTravassos** [comment]
ci:test:sf

File: portal-web/test/functional/com/liferay/portalweb/tests/enduser/collaboration/blogs/BlogsWidget.testcase [hunk 1]
```diff
@@ -463,13 +463,42 @@ definition {
...
+			entryContent = "Blogs Entry Content 2",
+			entryTitle = "Blogs Entry Title 2",
+			groupName = "Guest");
+
+		Navigator.gotoPage(pageName = "Blogs Page");
 
-		// TODO LPS-136811 CanViewEntryViaNavigationsPortlet pending implementation
+		CategoriesNavigationPortlet.selectCategoryPG(categoryName = "Category Name");
 
+		Blogs.viewEntryNotPresent(entryTitle = "Blogs Entry Content 1");
```
Comment (liuxingye) [inline portal-web/test/functional/com/liferay/portalweb/tests/enduser/collaboration/blogs/BlogsWidget.testcase:501]: @LeandroTravassos This is usually used for no entry assertion under blogs admin. We need to assert it on page in this case, so I use BlogsEntry.viewNoPG.

Comment (LeandroTravassos) [reply portal-web/test/functional/com/liferay/portalweb/tests/enduser/collaboration/blogs/BlogsWidget.testcase:501]: Ok. Thank you @liuxingye.

**Comment by liuxingye** [comment]
@LeandroTravassos 
Pull request submitted to https://github.com/austinchiang/liferay-portal/pull/1150
Please see my changes [here](https://github.com/austinchiang/liferay-portal/compare/5825460eb82f6295523d9944953d74c9b334e0d7...72d9310eb0682b903fddec298192ea1453782f51).

**Pull request closed without merging**